File: 2-semestre/uri-online-judge/javascript/1154.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let flag = false, idade, total = 0, contador = 0;

while (flag === false) {
    idade = parseInt(lines.shift());

    if (idade < 0) {
        flag = true;
    } else {
        total += idade;
        contador++;
    }
}

total /= contador;

console.log(`${total.toFixed(2)}`);

File: 2-semestre/uri-online-judge/c/1154.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    bool flag = false;
    int idades, contador = 0;
    double total = 0;

    while (flag == false) {
        scanf("%d", &idades);

        if (idades < 0) {
            flag = true;
        } else {
            total += idades;
            contador++;
        }
    }

    total /= contador;

    printf("%.2f\n", total);

    return 0;
}
